transfers/transfers.go

```diff
@@ -0,0 +1,67 @@
+package transfers
+
+import "github.com/checkout/checkout-sdk-go/common"
+
+const (
+	transfers = "transfers"
+)
+
+type TransferType string
+
+const (
+	Commission TransferType = "commission"
+	Promotion  TransferType = "promotion"
+	Refund     TransferType = "refund"
+)
+
+// Request
+type (
+	TransferSource struct {
+		Id     string `json:"id,omitempty"`
+		Amount int    `json:"amount,omitempty"`
```

Comment:
Is `int` correct here? what would happen with large number of amount?
I think we have a gap of amount between all the SDK

Reply:
I'm going to create a task so that we can verify this and change all the `int` amounts to `int64`